bin/storm.py

```diff
@@ -66,6 +66,12 @@ def init_storm_env():
     for option in options:
         value = config.get('environment', option)
         os.environ[option] = value
+        
+def get_java_cmd():
```

Comment:
Ah didn't know about the magic of Windows. That was why it worked anyway and I just wondered why it works. Thanks for letting me know. :)